### DISCUSSION TITLE
Mapbox gl js missing options

### DISCUSSION THREAD

**Discussion by ArtemisSaber** [DefinitelyTyped/DefinitelyTyped#63369] [Issues with a @types package]
Mapbox gl js supports worldview option in documents however the options type definition is missing that part for now.
[reference](https://docs.mapbox.com/mapbox-gl-js/api/map/)

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
It looks like this was fixed in #63174. Make sure you're using `@types/mapbox-gl` version 2.7.8 or later
